test-graalvm/build.gradle

```diff
@@ -28,7 +29,11 @@ subprojects { Project subproject ->
         return
     }
 
-    apply plugin: 'io.micronaut.application'
+    if (project.name == 'h2') {
```

Comment:
Ideally there shouldn't be an `if`, but both projects should apply a different plugin.